nemo/export/trt_llm/converter/model_converter.py

```diff
@@ -242,14 +267,12 @@ def model_to_trtllm_ckpt(
 
         if mapping.is_first_pp_rank():
             embedding_weight = (
-                np.ascontiguousarray(
-                    split(weights_dict["transformer.vocab_embedding.weight"], mapping.tp_size, mapping.tp_rank)
-                )
+                np.ascontiguousarray(split(weights_dict[vocab_embedding_key], mapping.tp_size, mapping.tp_rank))
```

Comment:
There are alot of `weights_dict[vocab_embedding_key]` I wonder if you could just reference the weight directly

Reply:
I agree, but since the PR is already quite big and there is an ongoing migration to Megatron, I'd leave it this way not to create further confusion.